websites/T/Trakt/presence.ts

```diff
@@ -0,0 +1,142 @@
+			presenceData.details = "The most watched shows";
+		else if (path.includes("/collected"))
+			presenceData.details = "The most collected shows";
+	} else if (path.includes("/movies")) {
+		try {
+			presenceData.details = "Browsing movie...";
+			presenceData.state = document
+				.querySelector(
+					"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1"
+				)
+				.textContent.split(
+					document.querySelector(
+						"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1 > span"
+					).textContent
+				)[0];
+		} catch (e) {
+			presenceData.details = "Browsing...";
+		}
+
+		if (path.includes("/trending")) presenceData.state = "Trending movies";
+		else if (path.includes("/popular")) presenceData.state = "Popular movies";
+		else if (path.includes("/recommended"))
+			presenceData.state = "Recommended movies";
+		else if (path.includes("/anticipated"))
+			presenceData.state = "Anticipated movies";
+		else if (path.includes("/watched"))
+			presenceData.state = "The most watched movies";
+		else if (path.includes("/collected"))
+			presenceData.state = "The most collected movies";
+		else if (path.includes("/boxoffice"))
+			presenceData.state = "Top 10 grossing movies";
+	} else if (path.includes("/calendars")) {
+		presenceData.details = "Browsing my calendar";
+		if (path.includes("/my")) {
+			if (path.includes("/shows")) presenceData.state = "Shows";
+			else if (path.includes("/premieres")) presenceData.state = "Premieres";
+			else if (path.includes("/new-shows")) presenceData.state = "New shows";
+			else if (path.includes("/movies")) presenceData.state = "Movies";
+			else if (path.includes("/dvd")) presenceData.state = "DVD & Blu-ray";
+		}
+		if (path.includes("/shows")) presenceData.state = "Shows";
+		else if (path.includes("/premieres")) presenceData.state = "Premieres";
+		else if (path.includes("/new-shows")) presenceData.state = "New shows";
+		else if (path.includes("/movies")) presenceData.state = "Movies";
+		else if (path.includes("/dvd")) presenceData.state = "DVD & Blu-ray";
+	} else if (path.includes("/discover"))
+		presenceData.details = "Finding new shows/movies...";
+	else if (path.includes("/apps")) presenceData.details = "Browsing apps";
+	else if (path.includes("/vip"))
+		presenceData.details = "Browsing VIP features";
+	else if (path.includes("/settings"))
+		presenceData.details = "Browsing settings";
+	else if (path.includes("/auth/signin"))
+		presenceData.details = "Registering on Trakt";
+	else if (path.includes("/auth/login"))
+		presenceData.details = "Logging in to Trakt";
+	else if (path.includes("/widgets")) presenceData.details = "Browsing widgets";
+	else if (path.includes("/search")) {
+		presenceData.details = "Searching for";
+		presenceData.state = new URLSearchParams(window.location.search).get(
+			"query"
+		);
+	}
```

Comment:
Please use a Record as shown here: https://github.com/PreMiD/Presences/blob/main/websites/T/TruckersMP/presence.ts#L15-L41 or use a sitch as that'll shorten the amount of lines here